Client/src/Components/DashboardSidebar/index.css

```diff
@@ -1,19 +1,19 @@
 .dashboard-sidebar {
-  height: 100vh;
+  height: calc(100vh - 64px);
   width: 250px;
   border: 1px solid var(--color-border-0);
-  border-radius: var(--border-radius-0);
```

Comment:
What was this border radius for?

Reply:
No idea tbh, it wasn't visible before the changes as the navbar was overlapped on top of it, so I decided to remove it.